src/seedu/addressbook/parser/Parser.java

```diff
@@ -22,6 +23,7 @@
 import seedu.addressbook.commands.ListCommand;
 import seedu.addressbook.commands.ViewAllCommand;
 import seedu.addressbook.commands.ViewCommand;
+import seedu.addressbook.commands.SortCommand;
```

Comment:
The import should also be sorted alphabetically. IIRC IDEs such as IntelliJ have auto import function that can do this for you :)

Reply:
Fixed. Thanks for the info :)